app/models/container_image.rb

```diff
@@ -38,9 +38,9 @@ class ContainerImage < ApplicationRecord
            :inverse_of => :resource
   has_one :last_scan_result, :class_name => "ScanResult", :as => :resource, :dependent => :destroy, :autosave => true
 
-  has_many :metric_rollups, :as => :resource, :dependent => :nullify, :inverse_of => :resource
-  has_many :metrics, :as => :resource, :dependent => :nullify, :inverse_of => :resource
-  has_many :vim_performance_states, :as => :resource, :dependent => :nullify, :inverse_of => :resource
```

Comment:
Not sure why these were nullify

Reply:
Yeah weird - in other models we just let those be orphaned, right?

Reply:
Correct, this is the only one that does nullify on metrics* and vim_perf*